README.md

```diff
@@ -1,17 +1,23 @@
 # eslint-config-formidable
-======================
+
 #### A composable set of ESLint configurations.
 [![Build Status](https://travis-ci.org/FormidableLabs/eslint-config-formidable.svg?branch=master)](https://travis-ci.org/FormidableLabs/eslint-config-formidable)
 [![npm](https://img.shields.io/npm/v/eslint-config-formidable.svg?maxAge=2592000)](https://www.npmjs.com/package/eslint-config-formidable)
 
-This project is based on [eslint-config-walmart](https://github.com/walmartlabs/eslint-config-walmart). It is `eslint@2+`-compatible and maintained by Formidable.
+This project is based on [eslint-config-walmart](https://github.com/walmartlabs/eslint-config-walmart) and maintained by Formidable. It supports ESLint 2, 3, and 4.
 
 ## Installation
 
-Install this config package and ESLint:
+Install ESLint and this config package:
+
+```bash
+$ npm install --save-dev eslint eslint-config-formidable
+```
+
+Depending on the [configutation](#usage) you choose, you'll most likely need one or more [additional eslint plugins](#dependencies) installed:
 
 ```bash
-$ npm install --save-dev eslint@2.10.2 eslint-config-formidable
+$ npm install --save-dev eslint eslint-config-<one> eslint-config-<two>
```

Comment:
NIT: Remove `eslint`. It's in the above command.

Reply:
oh dammit! yeah will do